code/modules/atmospherics/FEA_fire.dm

```diff
@@ -185,7 +186,25 @@
 #else
 			add_medium_light("hotspot", list(red, green, blue, 100))
 #endif
-
+	set_loc() // hotspots apparently start somewhere wacky
+		..()
+		src.update_overlay_blocking()
+		src.update_perspective_overlays()
```

Comment:
you should surround this and check if the loc is null, no use doing this stuff if we're in nullspace - that should resolve the !t issue below